### PR TITLE
[14.0] [MIG] sale_partner_incoterm

### PR DESCRIPTION
**Pull request by ReedHayashikawa**
sale_partner_incoterm migration from 13.0 to 14.0

### CODE REVIEW

**Comment by ReedHayashikawa** [comment]
Can you review this pull request?
@dreispt

File: sale_partner_incoterm/views/res_partner.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
         <field name="inherit_id" ref="stock.view_partner_stock_form" />
         <field name="arch" type="xml">
             <field name="property_payment_term_id" position="after">
-                <field name="customer_rank" invisible="1" />
+                <field name="customer_rank" attrs="{'invisible':1}" />
```
Comment (atchuthan) [inline sale_partner_incoterm/views/res_partner.xml]: Why do we need to change it from invisible to attrs?

File: sale_partner_incoterm/models/sale_order.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.onchange("partner_id")
+    def onchange_partner_id(self):
+        res = super().onchange_partner_id()
+        if not self.partner_id:
+            self.incoterm = False
+            return res
+        self.incoterm = self.partner_id.sale_incoterm_id
```
Comment (atchuthan) [inline sale_partner_incoterm/models/sale_order.py:13]: We don't need this code
```
        if not self.partner_id:
            self.incoterm = False
            return res
```
As when partner_id is not available then self.partner_id.sale_incoterm_id will result as False by default with the New ORM concept of recordset

**Review by atchuthan — CHANGES_REQUESTED** [review]
minor changes

File: sale_partner_incoterm/views/res_partner.xml (hunk 1 above)
Comment (dreispt) [inline sale_partner_incoterm/views/res_partner.xml]: This change is not needed, but not a blocker

**Review by dreispt — APPROVED** [review]

**Review by atchuthan — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
LGTM

**Pull request merged**